MMOCoreORB/src/server/zone/managers/resource/resourcespawner/ResourceSpawner.cpp

```diff
@@ -340,20 +361,41 @@ bool ResourceSpawner::writeAllSpawnsToScript() {
 
 			writer->writeLine("		},");
 
+			writer->writeLine("		inSpawn = \"" + String::valueOf(inPhase) + "\",");
```

Comment:
Out of curiosity, is the normal `resource_manager_spawns.lua` being used in the upload process?

If not, I'd encourage not changing its output format for compatibility with emu's upstream.